lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
Yeah mixing the two can be tricky. Honestly @MikeyBurkman could take one look at it and probably know better. Let me reach out to him and see.